Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "clap-permission-flag"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Could you leave the version out of this patch? That way we can change the version as part of the release PR (:

Reply:
Fixed in 8ccef1b